Documentation/dev-tools/kfence.rst

```diff
@@ -155,7 +163,7 @@ These are reported on frees::
     ==================================================================
     BUG: KFENCE: memory corruption in test_kmalloc_aligned_oob_write+0xef/0x184
 
-    Detected corrupted memory at 0xffffffffb6797ff9 [ 0xac . . . . . . ]:
+    Detected corrupted memory at 0xffffffffb6797ff9 [ 0xac . . . . . . ] (in kfence-#69):
```

Comment:
Same here about parentheses.

Also, this line and some others in KFENCE reports are quite long. I very much dislike when the lines of some kernel output get trimmed or split across multiple lines. Should we keep all report lines < 80 characters?

Reply:
In the case here we can do s/corrupted memory/corruption/, which should shorten this a bit. In the case here it'll become 77 chars with the parens removal.

Not sure what other lines are left. Trying to force all lines to be <80 might be hard though, and splitting this text across multiple lines is bad too because it makes grepping etc. impossible. Trying to balance readability, number of characters, and avoiding unnecessary lines requires trade-offs. I'm willing to go over 80 chars if there is no better option.

Reply:
I'm going with s/Detected c/C/ so it'll say "Corrupted memory at ...".